components/LinkButton/LinkButton.jsx

```diff
@@ -1,6 +1,6 @@
-import Link from "next/link";
-import { Box, Button, Text } from "@chakra-ui/react";
-import BackArrow from "../../public/images/back-arrow.svg";
+import Link from "next/link"
+import { Box, Button, Text } from "@chakra-ui/react"
+import BackArrow from "../../public/images/back-arrow.svg"
```

Comment:
It's not a huge problem (I'm no maniac for semi-colons), but I'm interested how they are disappearing?

Reply:
This semi-colons is from my vscode setting issue. I have fixed it now. New commit is up-todate with comented lines cleaned up.

Reply:
Thanks Amutha, makes the changes easier to read.

Reply:
I guess this is another reason to have something like Prettier set up on projects.